plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -1854,6 +1854,36 @@ Ufe::Path ProxyShape::ufePath() const
 }
 #endif
 
+//----------------------------------------------------------------------------------------------------------------------
+SdfPathVector ProxyShape::getTranslatedPrimPaths() const
+{
+  SdfPathVector paths(m_requiredPaths.size());
```

Comment:
Isn't this just
SdfPathVector paths;
paths.reserve(m_requiredPaths.size());
std::copy(m_requiredPaths.begin(), m_requiredPaths.end(), std::back_inserter(paths));
?